src/config/dynamic-theme-fixes.config

```diff
@@ -3967,6 +3982,34 @@ a[data-mentionid],
 
 ================================
 
+forums.laptopvideo2go.com
+
+INVERT
+.ipsReact_button
+
+CSS
+.ipsBadge {
+    --darkreader-bg--badge--background: var(--darkreader-neutral-background) !important;
+}
+.ipsReact_reactCount > a, 
```

Comment:
These 2 selectors can be added to the list of selectors under this line.

Reply:
@Gusted nope . ipsBadge need to be as is. 
![20210409-1617955527-001](https://user-images.githubusercontent.com/9846948/114149441-335f9180-991b-11eb-975c-9c59803cc7f7.png)

Reply:
I meant
```CSS
.ipsReact_reactCount > a,
.ipsReact_reactCount > span {
    background-color: var(--darkreader-neutral-background) !important;
}
```

As the one at line 4008 is the same declaration.

Reply:
This applied in latest commit fix.